### ISSUE TITLE
Style the new wins form

### ISSUE THREAD

**Issue by jbubar** [hackforla/website#1163]
### Dependency
#1060 

### Overview
Redesign the wins form. 
The current wins form is a google form, and @akibrhast wrote a really nice html form that outputs to a new google sheet. The new form will be on the site, but not linked. 
Could we redesign the new form?

### Action Items
- [ ] check out the new form
- [ ] check out the figma
- [ ] create a new design
- [ ] make some suggestions for how to change the functionality
- [ ] create a new issue for a dev to style it, and make it work the way you envision

### Resources/Instructions
[figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1209%3A0)
once dependency is done the new form will be at www.hackforla.org/share-your-wins/

**Comment by akibrhast** [comment]
Closing this because it is a duplicate of #1104
